contact.css

```diff
@@ -0,0 +1,105 @@
+body {
+    font-family: "Arial";
+    margin: auto;
+    background-image: url("IMG_1693.jpg");
+}
+
+#map {
+    height: 50%;
+    padding-bottom: 50%;
+    position: relative;
+    width: 90%;
+    margin: auto;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+input[type=text],
+textarea {
+    width: 90%;
+    padding: 12px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    resize: vertical;
+    margin: auto;
+}
+
+label {
+    padding: 12px 12px 12px 0;
+    display: inline-block;
+}
+
+
+/* This is the Submit button*/
+
+input[type=submit] {
+    background-color: black;
+    color: white;
+    padding: 12px 30px;
+    border: none;
+    border-radius: 4px;
+    cursor: pointer;
+    float: right;
+    margin-right: 80px;
+    margin-top: 30px;
+}
+
+input[type=submit]:hover {
+    background-color: #505350;
+}
+
+
+/*End of Submit button */
+
+.container {
+    border-radius: 5px;
+    background-color: #f2f2f2;
+    padding: 20px;
+    width: 90%;
+    margin: auto;
+}
+
+.col-20 {
+    float: left;
+    width: 25%;
+    margin-top: 6px;
+}
+
+.col-30 {
+    float: left;
+    width: 75%;
+    margin-top: 6px;
+}
+
+
+/* Clear floats after the columns */
+
+.row:after {
+    content: "";
+    display: table;
+    clear: both;
+}
+
+
+/* Responsive layout - when the screen is less than 600px wide, make the two columns stack on top of each other instead of next to each other */
+
+@media screen and (max-width: 600px) {
+    .col-20,
+    .col-30,
+    input[type=submit] {
+        width: 90%;
+        margin-top: 0;
+        margin: auto
+    }
+}
+
+.Biography {
+    background: none;
+    width: 90%;
+    margin: auto;
+    font-family: "Arial";
```

Comment:
You can remove this since you're already applying this to the entire body on `line 2`